### PR TITLE
Add simple integration test

### PR DESCRIPTION
**Pull request by calvinytong**
Starts the integration test process by resolving #87

### CODE REVIEW

**Comment by calvinytong** [comment]
@edublancas I think this is close to ready. The tests are failing because the examples do not work. I think we should fix the examples and then merge this so at least we don't break the examples again.

**Comment by calvinytong** [comment]
@jinhyunglee the example currently fails with 

```
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.5/bin/yass", line 11, in <module>
    load_entry_point('yass-algorithm', 'console_scripts', 'yass')()
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/travis/virtualenv/python3.5.5/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/travis/build/paninski-lab/yass/src/yass/command_line.py", line 54, in sort
    output_dir=output_dir, complete=complete)
  File "/home/travis/build/paninski-lab/yass/src/yass/command_line.py", line 63, in _run_pipeline
    pipeline.run(config, logger_level, clean, output_dir, complete)
  File "/home/travis/build/paninski-lab/yass/src/yass/pipeline.py", line 74, in run
    output_directory=output_dir)
  File "/home/travis/build/paninski-lab/yass/src/yass/process/run.py", line 309, in run
    CONFIG.clustering.masking_threshold,
  File "/home/travis/build/paninski-lab/yass/src/yass/config/config.py", line 62, in __getattr__
    return FrozenJSON(self._data[name])
KeyError: 'masking_threshold'
```

Is this an easy fix?

**Comment by calvinytong** [comment]
@jinhyunglee this is still failing. Can we get this fixed so we can merge?

**Comment by edublancas** [comment]
How are we going here? I know hooshmand's code for retinal evaluation works. It would be good to have a baseline to compare and add a section in the documentation for this. i.e. since I am refactoring the pipeline, I would like to know if I am doing still good in terms of accuracy so I would like to have some baseline to make sure that my changes aren't breaking the pipeline. can you help with this @calvinytong ?

**Comment by calvinytong** [comment]
As we discussed, this should go in as is. Just running the examples is our integration test. The evaluation code will be run on our server as cron job. I'm working on that but am currently blocked because our server has no storage.

**Comment by edublancas** [comment]
please ask the team if they can clean up space, I already did but I'm sure there are some stuff in the disk that we don't need anymore

**Comment by hooshmandshr** [comment]
Guys,

I cleaned up some space too, there is about 7G free now, but if we delete the rest of

**Comment by edublancas** [comment]
Thanks @hooshmandshr

**Comment by calvinytong** [comment]
Thanks! We should try to merge this soon. One more layer of protection

**Comment by edublancas** [comment]
I will merge this to `pipeline-split` branch. I am working on some refactoring and need this. Will also make sure to update examples and merge with master when everything works. Thanks calvin

**Comment by edublancas** [comment]
Since this is an old PR, there are many merge conflicts. Please update PR.

**Comment by calvinytong** [comment]
The merge conflict as just because it is rebased on master, it would be easier to put this into master and then have pipeline split rebase onto it

File: .travis.yml [hunk 1]
```diff
@@ -9,4 +9,5 @@ install:
   - make install
 script:
   - make test
+  - make integration-test
```
Comment (edublancas) [inline .travis.yml]: can you comment this thing? I will fix all the examples first

**Pull request merged**